remappers/hostmetrics/filesystem.go

```diff
@@ -0,0 +1,151 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package hostmetrics
+
+import (
+	remappers "github.com/elastic/opentelemetry-lib/remappers/internal"
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/pmetric"
+)
+
+// DeviceMetrics holds metrics data for a device
+type DeviceMetrics struct {
+	TotalUsage      int64
+	UsedBytes       int64
+	TotalInodeUsage int64
+}
```

Comment:
```suggestion
// DeviceMetrics holds metrics data for a device
type deviceMetrics struct {
	totalUsage      int64
	usedBytes       int64
	totalInodeUsage int64
}
```

I think we should not export this.